src/uct/ib/base/ib_device.c

```diff
@@ -1125,9 +1125,10 @@ ucs_status_t uct_ib_device_mtu(const char *dev_name, uct_md_h md, int *p_mtu)
     return UCS_OK;
 }
 
-int uct_ib_device_is_gid_raw_empty(uint8_t *gid_raw)
+int uct_ib_device_is_invalid_gid(uint8_t *gid_raw)
```

Comment:
IMO better make it a "positive" check
uct_ib_device_is_valid_gid

Reply:
also, the param can be const pointer

Reply:
done